submissions/7SergSmith7/Memory-pair-game/app.js

```diff
@@ -0,0 +1,105 @@
+import "../css/vendor/normalize.css";
+import "../css/styles.css";
+
+const images = [
+  "acrobatics",
+  "boneCutter",
+  "chaosRing",
+  "doubleStrike",
+  "innerDemon",
+  "metamorphosis",
+  "splittingSoul",
+  "manaBurn",
+];
+const pairCounter = 8;
+const winMsg = "You Win Bro! Play again?";
+const cards = [...images, ...images];
+const gameField = document.querySelector(".game-field");
+
+let firstCard = "";
+let secondCard = "";
+let pairsCounter = 0;
+
+function cardTemplate(image) {
+  return `<div class="card" data-name="${image}">
+            <div class="spin-card">
+              <div class="card-front">
+                <img src="./img/cardFront.png" class="front-img"/>
+              </div>
+              <div class="card-back">
+                <img class="back-img" src="./img/${image}.png" />
+              </div>
+            </div>
+          </div>`;
+}
+function shuffleCards(cardsArray) {
+  cardsArray.sort(() => {
+    return 0.5 - Math.random();
+  });
+}
+function renderNewField(cardsArray) {
+  shuffleCards(cardsArray);
+  let field = "";
+  cardsArray.forEach((img) => {
+    return (field += cardTemplate(img));
+  });
+  gameField.innerHTML = field;
+}
+
+gameField.addEventListener("click", onCardClick);
+
+function onCardClick(e) {
```

Comment:
You can destructure your event in the function argument as well
```suggestion
function onCardClick({target}) {
```